contracts/test/TestListingsRegistry.js

```diff
@@ -38,14 +31,14 @@ contract("ListingsRegistry", accounts => {
     await instance.create(ipfsHash, initPrice, initUnitsAvailable, {
       from: accounts[0]
     })
-    let listingCount = await instance.listingsLength()
+    const listingCount = await instance.listingsLength()
     assert.equal(
       listingCount,
       initialListingsLength + 1,
       "listings count has incremented"
     )
-    let [
-      listingAddress,
+    const [
+      ,
```

Comment:
Are these now rogue commas 😕

Reply:
@micahalcorn Nah, it's just destructuring syntax for ignoring values we don't care about. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment (section "Ignoring some returned values")